projects/cincinnati/OWNERS

```diff
@@ -0,0 +1,39 @@
+approvers:
+- aditya-konarde
+- crawford
+- jakedt
+- jfchevrette
+- jmelis
+- jottofar
+- jzelinskie
+- lalatendumohanty
+- lucab
```

Comment:
Luca removed himself from the Cincinnati OWNERS files recently, we can skip him here as well.

Reply:
Fixed